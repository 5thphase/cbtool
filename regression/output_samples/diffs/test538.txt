 status: Starting the attachment of vm_40, part of AI ai_5...
 status: Waiting for vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx), part of AI ai_5, to start...
 status: Starting the attachment of vm_39, part of AI ai_5...
 status: Waiting for vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx), part of AI ai_5, to start...
 status: Trying to establish network connectivity to vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx), part of AI ai_5, on IP address xxx.xxx.xxx.xxx ...
 status: Trying to establish network connectivity to vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx), part of AI ai_5, on IP address xxx.xxx.xxx.xxx ...
 status: Checking ssh accessibility on vm_40, part of AI ai_5 (ssh -p 22 klabuser@xxx.xxx.xxx.xxx )...
 status: Bootstrapping vm_40, part of AI ai_5 (creating file cb_os_paramaters.txt in "klabuser" user's home dir on xxx.xxx.xxx.xxx )...
 status: Sending a copy of the code tree to vm_40, part of AI ai_5 (xxx.xxx.xxx.xxx )...
 status: Checking ssh accessibility on vm_39, part of AI ai_5 (ssh -p 22 klabuser@xxx.xxx.xxx.xxx )...
 status: Bootstrapping vm_39, part of AI ai_5 (creating file cb_os_paramaters.txt in "klabuser" user's home dir on xxx.xxx.xxx.xxx )...
 status: Sending a copy of the code tree to vm_39, part of AI ai_5 (xxx.xxx.xxx.xxx )...
 status: Performing generic application instance post_boot configuration on all VMs belonging to ai_5...
 status: Running application-specific "setup" configuration on all VMs belonging to ai_5...
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1471276440 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1471276449 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1471276459 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1471276501 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1471276517 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|Redis Version |3.0.6 
|VMs APPLICATION ERRORS |6 
