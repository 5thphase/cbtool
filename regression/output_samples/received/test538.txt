 status: Waiting for vm_39 (cloud-assigned uuid 27BCD534-9618-5DE8-9185-1EC185E0BA8C) to start...
 status: Waiting for vm_40 (cloud-assigned uuid 27CA1720-300F-5FB4-9CCF-077C8B6C8EAF) to start...
 status: Trying to establish network connectivity to vm_39 (cloud-assigned uuid 27BCD534-9618-5DE8-9185-1EC185E0BA8C), on IP address 8.139.235.122...
 status: Trying to establish network connectivity to vm_40 (cloud-assigned uuid 27CA1720-300F-5FB4-9CCF-077C8B6C8EAF), on IP address 220.140.82.95...
 status: Bypassing the sending of a copy of the code tree to vm_39 (8.139.235.122)...
 status: Bypassing the sending of a copy of the code tree to vm_40 (220.140.82.95)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_5...
 status: Bypassing application-specific configuration on all VMs beloging to ai_5...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_5...
AI object FE826B58-2EC9-52E2-9AEA-D674F4039C3F (named "ai_5") sucessfully attached to this experiment. It is ssh-accessible at the IP address 8.139.235.122 (27BCD534-9618-5DE8-9185-1EC185E0BA8C.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |207.2.53.38     |1395353523     |none                                    |1AADB16A-B41A-52F5-B30D-CBF359AA9610
|ai_2      |hadoop         |1_x_hadoopmaster->3_x_hadoopslave                 |190.23.90.99    |1395353529     |none                                    |9B42A432-93C0-53CD-B179-CDAB75892684
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |181.56.245.120  |1395353541     |none                                    |A72B1E83-C201-5089-9B92-157CD1893D87
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |108.27.17.193   |1395353582     |none                                    |C333BEE4-D2FD-58BA-B817-E4B1639687A0
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |8.139.235.122   |1395353600     |none                                    |FE826B58-2EC9-52E2-9AEA-D674F4039C3F

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_26         |was                 |iron32      |253.205.77.148  |simhostb1                  |SUT                 |ai_1    |none       |539DFE00-C755-525C-90F3-86539ED3F5E4
|vm_27         |client_daytrader    |micro32     |207.2.53.38     |simhostd1                  |LG                  |ai_1    |none       |5E21D89A-A8CD-54D8-BD18-6AA9D8CF2321
|vm_28         |db2                 |platinum64  |20.182.23.38    |simhosta1                  |SUT                 |ai_1    |none       |76608BF1-A049-5947-909A-F11544DAFB5E
|vm_29         |hadoopmaster        |copper32    |190.23.90.99    |simhosta1                  |SUT                 |ai_2    |none       |3EDA7376-9FE5-5223-9BC5-2C587CD75BC1
|vm_30         |hadoopslave         |iron32      |62.35.218.125   |simhostc1                  |SUT                 |ai_2    |none       |E1697433-7D06-50D0-8E3E-4D5E1026FB28
|vm_31         |hadoopslave         |iron32      |170.55.145.108  |simhostc1                  |SUT                 |ai_2    |none       |70DBDB70-A0CC-5B3C-9423-03B4C8F377D3
|vm_32         |hadoopslave         |iron32      |120.38.6.150    |simhostb1                  |SUT                 |ai_2    |none       |F2117DE0-A1C2-559A-906E-DD9E71CA95AE
|vm_33         |client_daytrader    |micro32     |181.56.245.120  |simhostd1                  |LG                  |ai_3    |none       |C68F687A-306E-55B3-9A92-116BEDDDC6D9
|vm_34         |was                 |iron32      |67.49.117.55    |simhostb1                  |SUT                 |ai_3    |none       |A41C0C38-78EF-5F90-9560-F3AE87B090DE
|vm_35         |db2                 |platinum64  |80.210.230.78   |simhostb1                  |SUT                 |ai_3    |none       |C0631476-F237-5856-97F3-A5CF69BC3BFC
|vm_36         |was                 |iron32      |53.177.197.210  |simhosta1                  |SUT                 |ai_4    |none       |128B0B32-9BD0-5EC6-8286-E016CCE72818
|vm_38         |db2                 |platinum64  |78.84.205.217   |simhostb1                  |SUT                 |ai_4    |none       |B04ECA35-0E07-5E37-98D0-688E5E502DFB
|vm_37         |client_daytrader    |micro32     |108.27.17.193   |simhostd1                  |LG                  |ai_4    |none       |09D12748-308C-5AF5-884F-F6A6C316C16D
|vm_39         |netclient           |platinum64  |8.139.235.122   |simhosta1                  |SUT                 |ai_5    |none       |FE00547A-6FB6-5BC5-8878-8CEA1C547C3C
|vm_40         |netserver           |iron32      |220.140.82.95   |simhostb1                  |SUT                 |ai_5    |none       |94E510ED-05A1-52F9-B4D9-DDC3323D90DB

The AI object ai_5, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|detach_parallelism                      |20                                   
|ssh_key_name                            |bogus_id_rsa                         
|sut                                     |1_x_netclient->1_x_netserver         

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.11M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |30315                         
|Total Connections Received                           |1775                          
|Total Commands Processed                             |1815150                       
|Number of Keys                                       |524                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |2101248                       
|Data Size                                            |600704                        
|Index Size                                           |457856                        
|Average Object Size                                  |520.540727903                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |15                            
|AIs                                                  |5                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |15                            
|VMs ARRIVED                                          |37                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |5                             
|AIs ARRIVED                                          |5                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |155                           

 ---------------- VMC-WIDE COUNTERS ----------------
|F92423FA-5397-5C85-83AE-F1E571E97C48 (simzone_b) VM RESERVATIONS|6                             
|FF5DC0D8-1378-5755-B3BF-ACC9801512A9 (simzone_d) VM RESERVATIONS|3                             
|C2B34105-CEB6-53D4-9B01-6045EEB54595 (simzone_c) VM RESERVATIONS|2                             
|EC14CE9C-63F5-5D72-8982-32C59FE8ACE7 (simzone_a) VM RESERVATIONS|4                             

