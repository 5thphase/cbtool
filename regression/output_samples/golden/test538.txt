 status: Waiting for vm_39 (cloud-assigned uuid 89FE2612-F459-519E-B109-8563A3052F1A) to start...
 status: Waiting for vm_40 (cloud-assigned uuid 2F2424BB-CF26-5898-BFC7-C93AC9353661) to start...
 status: Trying to establish network connectivity to vm_39 (cloud-assigned uuid 89FE2612-F459-519E-B109-8563A3052F1A), on IP address 206.134.63.192...
 status: Trying to establish network connectivity to vm_40 (cloud-assigned uuid 2F2424BB-CF26-5898-BFC7-C93AC9353661), on IP address 38.253.175.127...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_39 (206.134.63.192)...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_40 (38.253.175.127)...
 status: Starting a new "performance emitter" for ai_5
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_5...
 status: Bypassing application-specific configuration on all VMs beloging to ai_5...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_5...
AI object 1088C26B-B699-5F92-86E0-4EABB7C21A2A (named "ai_5") sucessfully attached to this experiment. It is ssh-accessible at the IP address 208.192.169.1 (89FE2612-F459-519E-B109-8563A3052F1A.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |88.229.151.201  |1454604918     |none                                    |CA23D099-7677-5EFA-9919-B5E11033D1B0
|ai_2      |hadoop         |1_x_hadoopmaster->3_x_hadoopslave                 |58.73.88.131    |1454604925     |none                                    |280E79DE-79DE-54A9-A1FB-38937A9DA306
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |109.223.73.173  |1454604937     |none                                    |C211AAA2-A63B-5F87-9901-216AE5372952
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |28.7.122.40     |1454604982     |none                                    |18F6CE19-98A6-5FBF-8805-D4A23EDDB671
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |208.192.169.1   |1454604999     |none                                    |1088C26B-B699-5F92-86E0-4EABB7C21A2A

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_26         |client_daytrader    |micro32     |88.229.151.201  |simhostc4                  |LG            |private    |ai_1    |none       |C28CDCE8-E905-51B8-A380-FBD2DF29299F
|vm_27         |was                 |iron32      |108.162.29.19   |simhostc3                  |SUT           |private    |ai_1    |none       |1D966961-403E-5751-97DC-240032AC1A8F
|vm_28         |db2                 |platinum64  |114.30.204.122  |simhosta9                  |SUT           |private    |ai_1    |none       |4C6E8FC8-926F-5ABC-8D63-00C5C1EEC008
|vm_30         |hadoopslave         |platinum64  |144.114.129.38  |simhostb10                 |SUT           |private    |ai_2    |none       |C2E6D50B-8F3C-5737-8EEC-08E31227789D
|vm_29         |hadoopmaster        |platinum64  |58.73.88.131    |simhostc11                 |SUT           |private    |ai_2    |none       |41F2CD8F-9C04-58E5-A969-6079BFCE9672
|vm_32         |hadoopslave         |platinum64  |22.107.147.218  |simhostd2                  |SUT           |private    |ai_2    |none       |4F3E4E5D-737B-5043-A63E-47EC9544F6EF
|vm_31         |hadoopslave         |platinum64  |80.43.52.8      |simhosta0                  |SUT           |private    |ai_2    |none       |3401DBC9-9EF7-512B-B7C2-EA21780FAB6D
|vm_33         |client_daytrader    |micro32     |109.223.73.173  |simhostc9                  |LG            |private    |ai_3    |none       |498CA0EE-7E3D-5BA5-85FC-CEDE48215D55
|vm_34         |db2                 |platinum64  |207.241.137.254 |simhostc3                  |SUT           |private    |ai_3    |none       |489BBB5F-4261-5AC9-A003-1D1305F8F9C0
|vm_35         |was                 |iron32      |93.76.208.211   |simhostb2                  |SUT           |private    |ai_3    |none       |82EECCF2-B3ED-582F-9B46-A832FEE6083D
|vm_36         |client_daytrader    |micro32     |28.7.122.40     |simhostc6                  |LG            |private    |ai_4    |none       |648D9305-3E12-5731-82EB-BAA261B77E89
|vm_37         |db2                 |platinum64  |188.250.53.209  |simhostc4                  |SUT           |private    |ai_4    |none       |30A19C15-DE85-5216-8999-E0572AC219B5
|vm_38         |was                 |iron32      |80.57.170.8     |simhostd8                  |SUT           |private    |ai_4    |none       |693B746C-2876-5DF6-B864-1179CD1CD6B0
|vm_39         |netclient           |platinum64  |208.192.169.1   |simhostd4                  |SUT           |private    |ai_5    |none       |2FEE1078-D84E-5546-BD7B-05B00EC06385
|vm_40         |netserver           |iron32      |140.232.249.59  |simhosta8                  |SUT           |private    |ai_5    |none       |D5A26AD2-D940-5861-B307-7C45CA46022A

The AI object ai_5, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|detach_parallelism                      |20                                   
|ssh_key_name                            |cbtool_rsa                           
|sut                                     |1_x_netclient->1_x_netserver         

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |18.86M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12857143                      
|Total Connections Received                           |548706                        
|Total Commands Processed                             |42750679                      
|Number of Keys                                       |980                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |15                            
|AIs                                                  |5                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |15                            
|VMs ISSUED                                           |40                            
|VMs ARRIVED                                          |37                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |7                             
|AI RESERVATIONS                                      |5                             
|AIs ISSUED                                           |5                             
|AIs ARRIVED                                          |5                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |275                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9ADC8244-8C1F-5728-8581-9D8825A87B6E (simzone_a) VM RESERVATIONS|3                             
|662A8BAF-6270-52A0-B49B-7E1326CCC8B6 (simzone_c) VM RESERVATIONS|2                             
|8E167E6D-B4E1-531D-A90A-A469EF1067C6 (simzone_b) VM RESERVATIONS|7                             
|B37F024B-5D05-5130-8E9E-0804A1122684 (simzone_d) VM RESERVATIONS|3                             

